File: Code coach/Easy/Popsicles/popsicles.cpp
#include <iostream>
using namespace std;

int main()
{
	int siblings, popsicles;

	do
	{
		cin >> siblings >> popsicles;
	} while (siblings == 0);

	if (popsicles % siblings == 0)
	{
		cout << "give away";
	}
	else
	{
		cout << "eat them yourself";
	}

	return 0;
}

File: Code coaches/Easy/Popsicles/Popsicles.java
import java.util.Scanner; 
 
public class Popsicles { 

	public static void main(String[] args) { 
		Scanner input = new Scanner(System.in);
		int siblings;
		
		do {
			siblings = input.nextInt();
		} while(siblings == 0); 
			
		int popsicles = input.nextInt(); 
		
		if (popsicles % siblings == 0) {
			System.out.println("give away");
		} else {
			System.out.println("eat them yourself");
		}

		input.close();
	}
	
}